README.md

```diff
@@ -3,7 +3,7 @@
 </p> -->
 <h1 align="center">ECharts Java</h1>
 <p align="center">
-    <em>ECharts ❤️ Java = ECharts Java</em>
+    <em>"We bring better Vis into Java with ECharts"  ➡️  ECharts Java</em>
```

Comment:
Vis means Visualization?
Why use the abbreviation?

Reply:
The "➡️" symbol looks inconsistent. 
I think "We bring ... with ECharts" would be enough.